docs/source/plugin.rst

```diff
@@ -1,189 +1,128 @@
+      whole message (:term:`Match rule`), or any part of it
+      (:term:`Search rule`), or it may trigger for every matches in the message
+      (:term:`Find rule`).
+
+   Match rule
+      A match rule is a :term:`Generic rule` that triggers when the regex
+      matches the whole message.
+
+   Search rule
+      A search rule is a :term:`Generic rule` that triggers when the regex
+      matches any part of the message.
+
+   Find rule
+      A find rule is a :term:`Generic rule` that triggers for every time the
+      regex matches a part of the message.
+
+   Named rule
+      A named rule is a rule that uses a regular expression with a specific
+      format: with a name, and usualy with a prefix, or a set of specific
+      conditions. See :term:`Command`, :term:`Action command`, and
+      :term:`Nick command`. A named rule always matches the message from its
+      start, and it accepts any number of arguments::
+
+         [nick] <prefix><name> [<arg1> <arg2> <...> <argN>]
+
+      A named rule can have aliases, i.e. alternative names used to trigger
+      the rule.
+
+   Command
+      A command is a :term:`Named rule` that react to a :term:`Command prefix`
+      and a name.
+
+   Action command
+      An action command is a :term:`Named rule` that react to a name in a
+      message send with the ``ACTION`` intent/CTCP.
+
+   Nick command
+      A nick command (or nickname command) is a :term:`Named rule` that react
+      to a name prefixed by the bot's nickname in a message.
+
+   Command prefix
+      The command prefix is a regular expression joined to a :term:`Command`'s
+      name as its prefix. It is defined by configuration using
+      :attr:`core.prefix <sopel.config.core_section.CoreSection.prefix>`.
+
+   URL callback
+      A URL callback is a rule that triggers for every URL in a message.
+
+   Rate limiting
+      How often a :term:`rule <Rule>` can be triggered on a per-user basis, in
+      a channel, or across the IRC network.
+
+   Plugin callable
+      A plugin callable is a Python callable, that handles a specific message
+      from the IRC server matching a :term:`Rule`. See also the
+      :doc:`plugin/anatomy` chapter for the plugin callable signature.
+
+   Plugin job
+      A plugin job is a Python callable, that executes every given period of
+      time. See also the :doc:`plugin/anatomy` chapter for the plugin
+      job signature.
+
+   Single file plugin
+      A :term:`Sopel Plugin` composed of a single Python file. Can be loaded by
+      Sopel even when it's not available from ``sys.path``.
+
+   Folder plugin
+      A plugin composed of a directory that contains a ``__init__.py`` file.
+      It is not considered as a proper Python package unless it's parent
+      directory is in ``sys.path``. As it can create confusion, they are not
+      recommended, and either :term:`Single file plugin` or
+      :term:`Entry point plugin` should be used instead.
```

Comment:
```suggestion
      :term:`Entry point plugin` structure should be used instead.
```

Like? No?

Reply:
I feel like each "term" is the right label and it doesn't need any addition (like "structure" or "design").

So not a like, but I'm not an English expert, so I can miss some subtlety of the language.

Reply:
Fair enough. I was on the fence about this suggestion, and I don't like the added word "better enough" to push.